### PR TITLE
changes & add new source to madara

### PR DESCRIPTION
**Pull request by CVIUS**
- added lumine scans
- SBS: add nsfw override & change icon
- IS: change icon
- MK & MC: remove rate limit

### CODE REVIEW

File: multisrc/overrides/madara/mangaclash/src/MangaClash.kt [hunk 1]
```diff
@@ -1,22 +1,15 @@
...
-import eu.kanade.tachiyomi.lib.ratelimit.RateLimitInterceptor
 import eu.kanade.tachiyomi.multisrc.madara.Madara
 import java.text.SimpleDateFormat
 import java.util.Locale
-import okhttp3.OkHttpClient
-import java.util.concurrent.TimeUnit
 
 class MangaClash : Madara(
-    "MangaClash",
+    "Manga Clash",
```
Comment (CarlosEsco) [inline multisrc/overrides/madara/mangaclash/src/MangaClash.kt:8]: this will break the extension and cause people to have to migrate

File: multisrc/overrides/madara/mangakomi/src/MangaKomi.kt [hunk 2]
```diff
@@ -1,22 +1,15 @@
...
 class MangaKomi : Madara(
     "MangaKomi",
     "https://mangakomi.com",
     "en"
 ) {
 
-    override val client: OkHttpClient = super.client.newBuilder()
-        .addInterceptor(RateLimitInterceptor(1, 1, TimeUnit.SECONDS))
-        .build()
-
```
Comment (CarlosEsco) [inline multisrc/overrides/madara/mangakomi/src/MangaKomi.kt:19]: why did you remove rate limiter?

File: multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/madara/MadaraGenerator.kt [hunk 3]
```diff
@@ -118,7 +118,7 @@ class MadaraGenerator : ThemeSourceGenerator {
         SingleLang("InfraFandub", "https://infrafandub.xyz", "es"),
         SingleLang("Inmortal Scan", "https://manga.mundodrama.site", "es"),
         SingleLang("InstaManhwa", "https://www.instamanhwa.com", "en", isNsfw = true),
-        SingleLang("IsekaiScan.com", "https://isekaiscan.com", "en", className = "IsekaiScanCom", overrideVersionCode = 3),
+        SingleLang("IsekaiScan.com", "https://isekaiscan.com", "en", className = "IsekaiScanCom", overrideVersionCode = 4),
```
Comment (CarlosEsco) [inline multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/madara/MadaraGenerator.kt:121]: why is this being bumped?

**Review by CarlosEsco — CHANGES_REQUESTED** [review]

File: multisrc/overrides/madara/mangaclash/src/MangaClash.kt (hunk 1 above)
Comment (CVIUS) [reply multisrc/overrides/madara/mangaclash/src/MangaClash.kt:8]: Will revert.

File: multisrc/overrides/madara/mangakomi/src/MangaKomi.kt (hunk 2 above)
Comment (CVIUS) [reply multisrc/overrides/madara/mangakomi/src/MangaKomi.kt:19]: Will revert then.

File: multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/madara/MadaraGenerator.kt (hunk 3 above)
Comment (CVIUS) [reply multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/madara/MadaraGenerator.kt:121]: Changing icon

**Comment by CVIUS** [comment]
Will close this and will make another pr.
